package/src/custom/Extension/modules/Contacts/Ext/Vardefs/sugarfield_cause_of_death_c.php

```diff
@@ -0,0 +1,12 @@
+<?php
+ // created: 2018-01-31 21:21:45
+$dictionary['Contact']['fields']['cause_of_death_c']['labelValue']='Cause of Death';
+$dictionary['Contact']['fields']['cause_of_death_c']['full_text_search']=array (
+  'enabled' => '0',
+  'boost' => '1',
+  'searchable' => false,
```

Comment:
I'm curious why this is not searchable?

Reply:
I think custom fields are not searchable by default? This is generated by Studio.